src/instruction/aarch64_decode.C

```diff
@@ -28,37 +28,41 @@
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
  */
 
-#include "Instruction.h"
 #include "instruction_comp.h"
+#include "test_lib.h"
+
+#include "Instruction.h"
 #include "InstructionDecoder.h"
 #include "Register.h"
 #include "registers/aarch64_regs.h"
-#include "test_lib.h"
 
-#include <array>
-#include <boost/range/combine.hpp>
```

Comment:
these boost includes are unnecessary.

Reply:
it's showing the deleted lines, not the new lines that the diff shows replaced them.

Reply:
It looks like I butchered the manual fix-ups from the rebase. Thanks for checking closely.